cmd/osbuild-composer/main.go

```diff
@@ -21,7 +21,7 @@ var repositoryConfigs = []string{
 
 func main() {
 	var verbose bool
-	flag.BoolVar(&verbose, "v", false, "Print access log")
```

Comment:
can you comment this change in the commit message?

Reply:
Oh yea, that's the glog flag fiasco >.> will do!